wgsl/index.bs

```diff
@@ -12214,18 +12211,21 @@ WGSL provides two kinds of value constructors:
 
 Each [=type/concrete=], [=constructible=] *T* has a unique <dfn noexport>zero value</dfn>,
 and a corresponding built-in function written in WGSL as the type followed by an empty pair of parentheses: *T* `()`.
+[=Abstract numeric types=] also have zero values, but there are no built-in function to access them.
 
 The zero values are as follows:
 
 * `bool()` is `false`
-* `i32()` is 0
-* `u32()` is 0
-* `f32()` is 0.0
-* `f16()` is 0.0
+* `i32()` is 0i
```

Comment:
Use backticks on the constant to match bool? (here and below)

Reply:
We're not particularly consistent in how we do this in the spec. I'll file a separate issue to try and clean up the spec more generally in this regard. I am in favour of using code/backticks for actual values though.